source/advanced_components/docker/docker_integration.rst

```diff
@@ -0,0 +1,158 @@
++------------------------------+-----------------------------+------------------------+
+| `--opennebula-memory`        | `1024`                      | `ONE_MEMORY`           |
++------------------------------+-----------------------------+------------------------+
+| `--opennebula-template-id`   |                             | `ONE_TEMPLATE_ID`      |
++------------------------------+-----------------------------+------------------------+
+| `--opennebula-template-name` |                             | `ONE_TEMPLATE_NAME`    |
++------------------------------+-----------------------------+------------------------+
+| `--opennebula-network-id`    |                             | `ONE_NETWORK_ID`       |
++------------------------------+-----------------------------+------------------------+
+| `--opennebula-network-name`  |                             | `ONE_NETWORK_NAME`     |
++------------------------------+-----------------------------+------------------------+
+| `--opennebula-network-owner` |                             | `ONE_NETWORK_OWNER`    |
++------------------------------+-----------------------------+------------------------+
+| `--opennebula-image-id`      |                             | `ONE_IMAGE_ID`         |
++------------------------------+-----------------------------+------------------------+
+| `--opennebula-image-name`    |                             | `ONE_IMAGE_NAME`       |
++------------------------------+-----------------------------+------------------------+
+| `--opennebula-image-owner`   |                             | `ONE_IMAGE_OWNER`      |
++------------------------------+-----------------------------+------------------------+
+| `--opennebula-dev-prefix`    |                             | `ONE_IMAGE_DEV_PREFIX` |
++------------------------------+-----------------------------+------------------------+
+| `--opennebula-disk-resize`   |                             | `ONE_DISK_SIZE`        |
++------------------------------+-----------------------------+------------------------+
+| `--opennebula-b2d-size`      |                             | `ONE_B2D_DATA_SIZE`    |
++------------------------------+-----------------------------+------------------------+
+| `--opennebula-ssh-user`      | `docker`                    | `ONE_SSH_USER`         |
++------------------------------+-----------------------------+------------------------+
+| `--opennebula-disable-vnc`   | Enabled                     | `ONE_DISABLE_VNC`      |
++------------------------------+-----------------------------+------------------------+
+
+Using a template
+================================================================================
+
+Using a template means specifying either `--opennebula-template-id` or `--opennebula-template-name`. If you specify either of these two options, the following table applies, indicating what incompatible and what overrideable parameters are available:
```

Comment:
Maybe we should elaborate what this VM Template is for (although it is obvious it is better to give context)